apps/src/netsim/netsimGlobals.js

```diff
@@ -24,10 +31,12 @@ var netsim_ = null;
 module.exports = {
 
   /**
-   * Set the root controller that can be used for global operations.
+   * Set the root controllers that can be used for global operations.
+   * @param {StudioApp} studioApp
    * @param {NetSim} netsim
    */
-  setRootController: function (netsim) {
+  setRootControllers: function (studioApp, netsim) {
```

Comment:
Yes, it does.  It's called out explicitly here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/netsim.js#L185